api/lib/application/schooling-registration-dependent-users/index.js

```diff
@@ -37,6 +37,44 @@ exports.register = async function(server) {
         tags: ['api', 'schoolingRegistrationDependentUser']
       }
     },
+    {
+      method: 'POST',
+      path: '/api/schooling-registration-dependent-users/token',
+      config: {
+        auth: false,
+        handler: schoolingRegistrationDependentUserController.createAndReconcileUserToSchoolingRegistrationByToken,
+        validate: {
+          options: {
+            allowUnknown: false
+          },
+          payload: Joi.object({
+            data: {
+              attributes: {
+                'campaign-code': Joi.string().empty(Joi.string().regex(/^\s*$/)).required(),
+                'external-user-token': Joi.string().empty(Joi.string().regex(/^\s*$/)).required(),
+                'birthdate': Joi.date().format('YYYY-MM-DD').raw().required(),
+                'access-token': Joi.string().allow(null).optional(),
+              },
+              type: 'external-users',
+            },
+          }),
+          failAction: (request, h, err) => {
+            const errorHttpStatusCode = 400;
+            const jsonApiError = new JSONAPIError({
+              code: errorHttpStatusCode.toString(),
```

Comment:
C'est plutôt status au lieu de code.
```suggestion
              status: errorHttpStatusCode.toString(),
```
- status: the HTTP status code applicable to this problem, expressed as a string value.
- code: an application-specific error code, expressed as a string value.

Reply:
Done